modules/purge_purger_http/src/Form/ConfigurationForm.php

```diff
@@ -33,25 +40,79 @@ public function getFormID() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-    $form['test'] = [
+
```

Comment:
Although even I find it look nice that way, code in the method body (within {}) should be sticked straight to the "public function" line. In other words, delete line 43